src/engine/Collision/CollisionContact.ts

```diff
@@ -47,11 +47,11 @@ export class CollisionContact {
       this.normal = normal;
    }
 
-   resolve(delta: number, strategy: CollisionResolutionStrategy) {
```

Comment:
I may want to keep delta around for future improvements to resolution... but maybe we can cross that bridge when we come to it